src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements Area, Drawable {
+    protected Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+}
```

Comment:
It's good practice to override `toString()` in your `Figure` class to provide a string representation of the object, especially since you have a `color` attribute that could be included in the output.